guide/designing-products/lightning-services.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: guide
+title: Lightning Services
+description: Lightning services and their role in Bitcoin applications.
+nav_order: 10
+parent: Designing Bitcoin products
+permalink: /guide/designing-products/lightning-services/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources
+
+-->
+
+# Lightning services 
+
+A Lightning service is offered by some Bitcoin applications to avoid users having to manually manage and configure their apps. These services should be non-custodial and as trust-minimized as possible.
+
+Applications that offer Lightning services are often referred to as Lightning Service Providers (LSPs). Much like an Internet Service Provider (ISP) that offers internet based services LSPs offer, mostly Lightning, Bitcoin services. Roy Sheinfield from Breez elaborates further in his piece on [Envisioning LSPs in the Lightning Economy](https://medium.com/breez-technology/envisioning-lsps-in-the-lightning-economy-832b45871992).
+
+When considering offering Lightning services in your application you should consider the below questions:
+
+* What Lightning services, if any, will your application offer and why?
+* What potential liabilities does your application take on with these services?
+* What trade-offs are being made with the Lightning services you are offering and how will they be communicated to users? 
+
+Lightning services and LSPs are still an early development though the UX improvements they are currently offering can not be understated.
+
+Below are some popular Lightning services used today. 
+
+## Pay-to-open  
+
+Pay-to-open is when a user pays to have a channel with inbound (receiving) liquidity opened when they receive a payment. A channel is only opened if the receiver has no current channels with inbound liquidity, say when funding a new wallet, or their current channels do not have enough inbound liquidity to receive the payment. This is also referred to as ‘on-demand’ channel opening. 
+
+#### When to use
+
+* When users of your application are not experienced enough to manage their own payment channels.
+* If your application runs mobile lightning nodes that do not operate a dedicated 24/7 Lightning node.
+* If you want your users to fund a wallet application for the first time directly with a Lightning payment. 
+
+#### Trade-offs
+
+* Receiver needs to connect to the service provider's LSP server which has privacy implications.
```

Comment:
@Bosch-0 When you say "service provider", what do you mean? I ask because unpacking the acronym results in "the service provider's Lightning service provider", which is quite a mouthful and a little confusing.